packages/kas/src/nodes.ts

```diff
@@ -293,7 +330,7 @@ class Expr {
 
     // raise this expression to a given exponent
     // most useful for eventually implementing i^3 = -i, etc.
-    raiseToThe(exp) {
+    raiseToThe(exp: Expr, options?: {preciseFloats?: boolean}): Expr {
```

Comment:
Same here: do we need `options` if it's never used? Or do we need it in order to tell TS that this is implementing an abstract class or something?

Reply:
We need it here because one of the subclasses uses it.